pyansys_sphinx_theme/static/css/theme.css

```diff
@@ -179,3 +179,180 @@ p.rubric {
     margin-top: 0.75em;
     margin-bottom: 0.75em;
 }
+
+
+
+/*
+###############
+   table 
+###############
+*/
+
+table {
+  width:100%;
+  max-width: 100%; 
+  border-spacing: 0;
+  border-collapse: collapse;
+  overflow: hidden;
+  vertical-align: middle;
+
+  /* Disabling scroll bars */
+  overflow-y: scroll;
+  scrollbar-width: none; /* Firefox */
+  -ms-overflow-style: none;  /* Internet Explorer 10+ */
+  }
+
+tr { 
+  background-color: white;
+}
+
+th {
+  background-color: rgb(255, 183, 27, 0.55);
+}
+
+tr:nth-child(odd){ 
+  background-color: white;
+}
+
+tr:nth-child(even){
+background-color: rgb(255, 183, 27, 0.15); 
+}
+
+/*
+###############
+   longtable 
+###############
+
+Use as:
+
+.. table:: 
+  :class: table-centered
+
+  +---------------------------+-------------------+
+  |                           | MAPDL Command     |
+  +===========================+===================+
+  | **GUI commands**          | * ``*ASK``        |
+  |                           +-------------------+
+  |   ...
+  |   ...
+
+
+*/
+table.table-centered {
```

Comment:
This block looks the same as the `table` above, is it possible to merge them?

Reply:
I forgot to add ``text-align: centered`` to this block.